mne/epochs.py

```diff
@@ -1209,7 +1212,7 @@ def _compute_aggregate(self, picks, mode="mean"):
                 n_events += 1
 
             if n_events > 0:
-                data = np.nanmean(data)
+                data /= n_events
```

Comment:
I am still a bit confused about this part of the function, to be honest, maybe a comment would be good at that point

Reply:
this whole stuff only happens in the case of not-preloaded data, and thats why i guess it *has* to be somehow hacky and hard to follow ;) 
in this case there is no numpy array of data in memory that we can simply access and change (probably only a view or so..)
so they create this mock `data` object, fill it up by some loop through the epochs object and then average that.

but that's also why it's not really relevant for us - i doubt that we could even set not-preloaded data to `nan` easily without larger changes (i assume this info would have to be stored in some meta data and then applied when finally loading at some later point?), so it doenst make sense to implement an averaging function for this case.

Reply:
not super sure about the second part of my reply, maybe it is easier than i think. still, if we ask data to be preloaded, it doesnt apply

Reply:
okey, thank you for the explanation, now I finally understand what is going on.